### DISCUSSION TITLE
<br/> tag in table content not work when render as docx

### DISCUSSION THREAD

**Discussion by hanchenduming** [quarto-dev/quarto-cli#10045] [Q&A] [answered]
````qmd
### Description

---
format:
  docx:
    reference-doc: ./templates_now.docx
    toc: false
    number-sections: false
    highlight-style: github
    tbl-colwidths: [ 12,23,17,17,31 ]
---

# Login

| **No** |                        **Class**                        | **Desc** | **ToRun** |                   **Detail**                    |
|:------:|:-------------------------------------------------------:|:--------:|:---------:|:-----------------------------------------------:|
|   1    | com.github.<br/>repo.web.auth.service.<br/>LoginService |  Login   |   run()   | 1. login<br/>2. logout<br/>3. record login info |

````

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.

This being said, why would you expect HTML markups to work in Word?


Also, for accessibility reasons, please state the issue as plain text in the body of your post. Setting a title is not enough. Thanks for your understanding.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by hanchenduming** [reply]
Sorry for that, I have edited, I just need a line break in table content ， I was using Markdown before

**Reply by mcanouil** [reply]
You need grid tables not pipe table as documented in https://quarto.org/docs/authoring/tables.html#grid-tables

**Reply by cderv** [reply, marked as answer]
Example of table that would work 

````markdown
# Login

+---------+-------------------------+----------+-----------+----------------------+
| **No**  | **Class**               | **Desc** | **ToRun** | **Detail**           |
+:=======:+:=======================:+:========:+:=========:+:====================:+
| 1       | com.github.\            | Login    | run()     | 1. login            |
|         | repo.web.auth.service.\ |          |           | 2. logout           |
|         | LoginService            |          |           | 3. record login info |
+---------+-------------------------+----------+-----------+----------------------+

````

You can use Visual Editor to help you format table for example in [VS CODE](https://quarto.org/docs/visual-editor/vscode/) or [RStudio IDE](https://quarto.org/docs/visual-editor/)

With Quarto, when you need some complex formatting which is not always easy to express with Markdown syntax (because of variants and others stuff), you can use HTML syntax as Quarto will parse the HTML table (https://quarto.org/docs/authoring/tables.html#html-tables) 

Hope this additional informations help understand
